packages/alea-frontend/components/lo-explorer/index.tsx

```diff
@@ -0,0 +1,619 @@
+  setChosenArchives: React.Dispatch<React.SetStateAction<string[]>>;
+}) {
+  const handleArchiveChange = (event: React.ChangeEvent<HTMLInputElement>, newValue: string[]) => {
+    setChosenArchives(newValue);
+  };
+
+  return (
+    <FormControl fullWidth sx={{ flex: 1, minWidth: '250px' }}>
+      <Autocomplete
+        multiple
+        options={uniqueArchives}
+        value={chosenArchives}
+        limitTags={1}
+        onChange={handleArchiveChange}
+        renderInput={(params) => <TextField {...params} label="Archives" />}
+        renderOption={(props, option, { selected }) => {
+          const { icon } = getUrlInfo(
+            uniqueArchiveUris.find((uri) => getUrlInfo(uri).projectName === option)
+          );
+          return (
+            <li {...props}>
+              <Checkbox checked={selected} />
+              <ListItemText primary={option} />
+              {icon}
+            </li>
+          );
+        }}
+        renderTags={(value, getTagProps) =>
+          value.map((selectedOption, index) => {
+            const uri = uniqueArchiveUris.find(
+              (uri) => getUrlInfo(uri)?.projectName === selectedOption
+            );
+            const { icon } = getUrlInfo(uri) || {};
+            return (
+              <Chip
+                key={selectedOption}
+                label={
+                  <Box display="flex" alignItems="center" gap={1}>
+                    {icon}
+                    <Typography variant="body2">{selectedOption}</Typography>
+                  </Box>
+                }
+                {...getTagProps({ index })}
+              />
+            );
+          })
+        }
+        sx={{ flex: '1 ' }}
+        disableCloseOnSelect
+        fullWidth
+      />
+    </FormControl>
+  );
+}
+
+export function LoExplorer() {
+  const [searchString, setSearchString] = useState('');
+  const [chosenRelations, setChosenRelations] = useState<AllLoRelationTypes[]>([]);
+  const [chosenConcepts, setChosenConcepts] = useState<string[]>([]);
+  const [chosenLoTypes, setChosenLoTypes] = useState<LoType[]>([]);
+  const [chosenArchives, setChosenArchives] = useState<string[]>([]);
+  const [chosenArchivesUris, setChosenArchivesUris] = useState<string[]>([]);
+  const [loUris, setLoUris] = useState<Record<LoType, string[]>>({
+    definition: [],
+    problem: [],
+    example: [],
+    para: [],
+    statement: [],
+  });
+  const [selectedUri, setSelectedUri] = useState('');
+  const [selectedLo, setSelectedLo] = useState<LoType>('problem');
+  const [cart, setCart] = useState<CartItem[]>([]);
+  const [showCart, setShowCart] = useState(false);
+  const [isSearching, setIsSearching] = useState(false);
+  const { mmtUrl } = useContext(ServerLinksContext);
+  const [conceptsList, setConceptsList] = useState<Record<string, string>>({});
+  const [uniqueArchives, setUniqueArchives] = useState<string[]>([]);
+  const [uniqueArchiveUris, setUniqueArchiveUris] = useState<string[]>([]);
```

Comment:
`uniqueArchives` and `uniqueArchiveUris` is derivable state. You should not use `useState` for variables that can be derived directly from other variables.

It is okay to compute them on every render if the computation is not computationally expensive. Use `useMemo` if you would like to avoid unnecessary computation.
See `filteredUris` below

Reply:
Check all state variables in LOExplorer and update derived state variables, if needed